ush/parsing_namelists_CICE.sh

```diff
@@ -70,7 +70,7 @@ local CICE_RESTART_FORMAT="pnetcdf2"
 local CICE_DUMPFREQ="y"  # "h","d","m" or "y" for restarts at intervals of "hours", "days", "months" or "years"
 local CICE_DUMPFREQ_N=10000  # Set this to a really large value, as cice, mom6 and cmeps restart interval is controlled by ufs.configure
 local CICE_DIAGFREQ=$(( 86400 / DT_CICE ))  # frequency of diagnostic output in timesteps, recommended for 1x per day
-local CICE_HISTFREQ_N="0, 0, ${FHOUT_ICE}, 1, 1"
+local CICE_HISTFREQ_N="0, 0, ${FHOUT_ICE}, 0, 1"
```

Comment:
What is the purpose of this change?

Reply:
see https://github.com/NOAA-EMC/global-workflow/issues/3121

Reply:
OK, thanks.  Could you update the title of this PR and the description?  Also, please include the issues that this PR will close in description as
Resolves #3121 
Resolves #2985